scripts/backtest_uk_gsp.py

```diff
@@ -193,21 +186,19 @@ def get_available_t0_times(start_datetime, end_datetime, config_path):
     return available_init_times
 
 
-def get_loctimes_datapipes(config_path):
-    """Create location and init-time datapipes
+def get_times_datapipe(config_path):
+    """Create init-time datapipe
 
     Args:
         config_path: Path to data config file
 
     Returns:
-        tuple: A tuple of datapipes
-            - Datapipe yielding locations
-            - Datapipe yielding init-times
+        Datapipe: A Datapipe yielding init-times
     """
 
     # Set up ID location query object
     ds_gsp = get_gsp_ds(config_path)
-    gsp_id_to_loc = GSPLocationLookup(ds_gsp.x_osgb, ds_gsp.y_osgb)
+    GSPLocationLookup(ds_gsp.x_osgb, ds_gsp.y_osgb)
```

Comment:
I might be missing something here, is this missing an assignment to a variable or does it do something else? Can't see it referenced elsewhere

Reply:
Nope, you're right. That was just a blunder